### ISSUE TITLE
📑 Sidebar: combine file Activity and Comments into »Activity« timeline tab (possibly also "Versions")

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#658]
Basically as written up and with mockups at https://github.com/owncloud/core/issues/20259
We need to finally streamline the sidebar instead of putting more and more stuff in there.
![sidebar-mockup](https://cloud.githubusercontent.com/assets/925062/17250796/40602fce-55a6-11e6-9802-9f6853ab3923.png)

cc @nextcloud/javascript @rullzer @MorrisJobke @schiessle

**Comment by jancborchardt** [comment]
@danxuliu I just assigned you since you wanted to check it out. :) Let me know if there’s any questions or update.

**Comment by MorrisJobke** [comment]
Push to 15?

**Comment by jancborchardt** [comment]
@MorrisJobke hopefully :D – at this point already the Android app is doing this properly.

**Comment by juliushaertl** [comment]
@skjnldsv Maybe we can approach this in the same run, when we plan to do the sidebar work for 16? I'd say we discuss that in detail at the hackweek.

**Comment by georgehrke** [comment]
@juliushaertl @skjnldsv Are there plans to move the sidebar to Vue?

**Comment by skjnldsv** [comment]
@georgehrke To move to something we can master, yes! :)
We really haven't yet talked about the specifications!

See #10289

**Comment by jancborchardt** [comment]
@georgehrke that’s ok! :) Don’t despair.

@juliushaertl @georgehrke @skjnldsv and yes – let’s have some good discussion – and work – about the sidebar at the hackweek! :) We already defined a lot, it’s time we dive into the nitty gritty.

**Comment by jancborchardt** [comment]
@georgehrke heya, any status update on this? :) Or what are the difficulties?

**Comment by georgehrke** [comment]
@jancborchardt This was supposed to be based on #10289, which was postponed to 17

**Comment by jancborchardt** [comment]
Does it need to be blocked by that? It seems like separate things, and I’d wager the standardization will take longer …

**Comment by juliushaertl** [comment]
@georgehrke The implementation can happen independent from the standardization. Ove we have a vue component for the merged timeline, we can just plug it in either to the existing backbone based sidebar or to the standardized one, once ready. :wink:

**Comment by tobiasKaminsky** [comment]
@marinofaggiana comments and activities should be combined in "activity" tab, like on Android.

**Comment by tobiasKaminsky** [comment]
Regarding server we currently have five tabs which are too much. 
Let's at least for now combine comments and activity into one "activity" tab. @nextcloud/server-triage

**Comment by jancborchardt** [comment]
Wow amazing, @susnux great job on delivering one of the more long-standing design proposals. :partying_face: Really is much nicer with 4 tabs instead of 5 in the sidebar now. :)
